packages/host/app/lib/sqlite-adapter.ts

```diff
@@ -226,7 +232,10 @@ export default class SQLiteAdapter implements DBAdapter {
       .replace(/\.text_value/g, '.value')
       .replace(/\.jsonb_value/g, '.value')
       .replace(/= 'null'::jsonb/g, 'IS NULL')
-      .replace(/COLLATE "POSIX"/g, '');
+      .replace(/COLLATE "POSIX"/g, '')
+      .replace(/array_agg\(/g, 'json_group_array(')
+      .replace(/array_to_json\(/g, 'json(')
+      .replace(/jsonb_array_unnested/g, 'json_each.value'); // jsonb_array_unnested is a custom column reference in the SQL (not part of SQL standard) that you can use to reference the value of the jsonb_array_elements_text function (used in searchPrerendered)
```

Comment:
This is a pg/sqlite compatibility solution that I don't really love... but struggling to see how it could be done in a better way.  So one of the CTE in our query looks like this:

```
WITH instance_deps AS (
  SELECT 
    file_alias AS instance_file_alias,
    jsonb_array_unnested AS dep_url_file_alias - this needs to change for sqlite 
  FROM boxel_index,
    jsonb_array_elements_text(deps) AS jsonb_array_unnested -- this needs to change for sqlite 
  WHERE ...
),
```

And for this to work in SQLite, we need to transform it to this:

```
WITH instance_deps AS (
  SELECT
    file_alias AS instance_file_alias,
    json_each.value AS dep_url_file_alias -- this changed (not trivial to adjust because how do you know which column reference to change to json_each.value?)
  FROM boxel_index i,
    json_each(deps)  -- this is changed (trivial adjustment, use different function and remove alias)
   WHERE...
```

I opted into a solution that requires the developer to know using `jsonb_array_unnested` as a column reference for `json_each` will change to `json_each.value` in the sqlite adjuster.

Reply:
We already have `jsonb_array_elements_text` that does a very similar thing. Does that not work for you?

Reply:
That replaces `jsonb_array_elements_text` with `json_each` which is ok but my case also needs `json_each.value` to be used in lieu of the column reference (this is noted above in the "not trivial" case)

Reply:
ok, so in this case are we claming the "json_each" namespace (virtual column name) in the predicate of the query? Like does this assume you are not using an `AS` with `json_each`, and getting the default "column name" `json_each` in the `WHERE` part of your query? if so that seems brittle. like why can't you just name with `AS`, like 
```sql
SELECT json_each(deps) as deps_each FROM boxel_index WHERE deps_each.value = 'blah';
```

Then I think you could use the jsonb_array_elements, right? where instead of saying `deps_each.value` in your predicate you can say `deps_each.jsonb_value` which gets converted to `deps_each.value` from the regex above. does this not work?